src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.ts

```diff
@@ -135,7 +135,11 @@ export class TutorExerciseDashboardComponent implements OnInit {
                 }
 
                 this.getSubmissions();
-                this.getSubmissionWithoutAssessment();
+
+                // We don't want to assess submissions before the exercise due date
+                if (this.exercise.dueDate.isBefore(Date.now())) {
```

Comment:
We should additionally check, if the due date is set. Otherwise we will get a "Cannot read property 'isBefore' of null" error, if it is not.